ownCloud/Client/Viewer/DisplayViewController.swift

```diff
@@ -543,6 +543,12 @@ extension DisplayViewController {
 		self.core = configuration.core
 		self.state = configuration.state
 	}
+
+	func willLooseFocus() {
+	}
+
+	func willBecomeFocus() {
```

Comment:
@hosy may be `willBecomeFocused()` would be just more grammatically correct? Also... is it not something already implemented by the responder chain stuff? All `UViewControllers` are actually subclasses of `UIResponder`.

Reply:
You are right and now I´m using `viewDidAppear` and `viewDidDisappear` instead. Before I thought this will not work, when app is going to background. Thank you for that hint! Please give it a new try with the latest commit.